src/test/mediapipe/python/pythonNodeFinalizeException.py

```diff
@@ -0,0 +1,25 @@
+#*****************************************************************************
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+
+class OvmsPythonModel:
+    def initialize(self, kwargs: dict):
+        return True
+
+    def execute(self, inputs: list, kwargs: dict) -> list:
+        return None
+    
+    def finalize(self):
+        return NotExistingException
```

Comment:
I know it doesn't matter since it throws anyway, but it's still a bit confusing. Please change to
`return not_existing_variable` or `raise NotExistingException()`